arangod/GeneralServer/VstCommTask.cpp

```diff
@@ -290,11 +290,11 @@ void VstCommTask::handleAuthHeader(VPackSlice const& header, uint64_t messageId)
     LOG_TOPIC("01f44", ERR, Logger::REQUESTS) << "Unknown VST encryption type";
   }
 
-  auto entry = _auth->tokenCache().checkAuthentication(_authMethod, authString);
-  _authorized = entry.authenticated();
+  _authToken = _auth->tokenCache().checkAuthentication(_authMethod, authString);
+  _authorized = _authToken.authenticated();
 
   if (_authorized || !_auth->isActive()) {
-    _authenticatedUser = std::move(entry._username);
+    _authenticatedUser = std::move(_authToken._username);
```

Comment:
same

Reply:
argument from above is valid here. going to fix it.